### DISCUSSION TITLE
Problem with dependency update.

### DISCUSSION THREAD

**Discussion by namespaceahr** [open-telemetry/opentelemetry-java-instrumentation#6795] [Q&A] [answered]
Hello everyone,  I am trying to update this dependency to 1.18.0-alpha. It seems the latest version is 1.16.0-alpha. The main problem with v1.16.0-alpha is that it brings other dependencies versions to 1.16.0-alpha. So I can't update other dependencies. Is there any changes in the artifact? Any suggestions would be appreciated.

<groupId>io.opentelemetry.instrumentation</groupId>
<artifactId>opentelemetry-otlp-exporter-starter</artifactId>
<version>1.16.0-alpha</version>

**Comment by mateuszrzeszutek** [comment, marked as answer]
Hey @namespaceahr ,
Since version 1.17.0, the OTLP exporter is now included in the `opentelemetry-spring-boot-starter` module. You can safely remove the `opentelemetry-otlp-exporter-starter` from your build file.
